packages/protocol/contracts/shared/bridge/Bridge.sol

```diff
@@ -595,8 +595,7 @@ contract Bridge is EssentialResolverContract, IBridge {
         if (_message.to == address(_signalService)) return true;
 
         return _message.data.length >= 4
-            && bytes4(_message.data) != IMessageInvocable.onMessageInvocation.selector
-            && _message.to.isContract();
```

Comment:
And looks like:
```solidity
using Address for address;
using LibAddress for address payable;
```
are also not used in `Bridge.sol` anymore, shall we remove them too in this PR?